src/api/java/com/minecolonies/api/entity/ai/citizen/guards/GuardGearBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.minecolonies.api.entity.ai.citizen.guards;
+
+import com.minecolonies.api.util.constant.ToolType;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.util.Tuple;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GuardGearBuilder
```

Comment:
This is not what I mean with a builder class, but okey……………. Try have IntelliJ generate one for you.